keyboards/cospad/keymaps/johannbl/keymap.c

```diff
@@ -0,0 +1,72 @@
+#include QMK_KEYBOARD_H
+
+/* THIS FILE WAS GENERATED!
+ *
+ * This file was generated by qmk-compile-json. You may or may not want to
+ * edit it directly.
+ */
+
+enum custom_keycodes {
+  DOUBLEZERO = SAFE_RANGE,
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case DOUBLEZERO:
+      if (record->event.pressed) {
+        // when keycode DOUBLEZERO is pressed
+        SEND_STRING("00");
+      } else {
+        // when keycode DOUBLEZERO is released
+      }
+      break;
+
+  }
+  return true;
+};
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+        /* Layer 0
+ 		* ,-------------------.
+ 		* |Mute|Vol-|Vol+| FN |
+ 		* |----|----|----|----|
+ 		* |Tab | /  | *  | BS |
+ 		* |----|----|----|----|
+ 		* | 7  | 8  | 9  | -  |
+ 		* |----|----|----|----|
+ 		* | 4  | 5  | 6  | +  |
+ 		* |----|----|----|----|
+ 		* | 1  | 2  | 3  |    |
+ 		* |----|----|----| En |
+ 		* | 00 | 0  | .  |    |
+ 		* `-------------------'
+ 		*/
+        [0] = LAYOUT_numpad2_6x4(
+        	KC_MUTE, KC_VOLD, KC_VOLU, TT(2), 
+        	KC_TAB, KC_PSLS, KC_PAST, KC_BSPC, 
+        	KC_P7, KC_P8, KC_P9, KC_PMNS, 
+        	KC_P4, KC_P5, KC_P6, KC_PPLS, 
+        	KC_P1, KC_P2, KC_P3,
+        	DOUBLEZERO, KC_P0, KC_PDOT, KC_PENT
+        ),
+        
+        [1] = LAYOUT_numpad2_6x4(
+        	RGB_TOG, RGB_RMOD, RGB_MOD, KC_TRNS, 
+        	KC_NO, RGB_SPD, RGB_SPI, KC_NO, 
+        	RGB_M_G, RGB_M_SW, RGB_M_K, RGB_M_SN, 
+        	RGB_M_X, RGB_M_R, RGB_M_P, RGB_M_B, 
+        	RGB_HUI, RGB_SAI, RGB_VAI, 
+        	RGB_HUD, RGB_SAD, RGB_VAD, RESET
+        ),
```

Comment:
How is this layer activated?

Reply:
Good point. I can't say the keymap file is very clean. This is the one I use at the moment and I don't care about backlighting. I could instead propose 2 more standard numpad keymap. One for each layout.